fmriprep/workflows/confounds.py

```diff
@@ -243,3 +262,197 @@ def reverse_order(inlist):
     if isinstance(inlist, list):
         inlist.reverse()
     return inlist
+
+
+def get_ica_confounds(ica_out_dir):
```

Comment:
Hi @maartenmennes! We are trying to add ICA AROMA to FMRIPREP, but without explicitly denoising the data - only estimating noise regressors and giving them to the users to include in their design. Could you have a quick look at how we calculate residuals for non aggressive components? Much obliged!